themes/bootstrap3/js/lightbox.js

```diff
@@ -89,7 +98,14 @@ VuFind.lightbox = (function() {
           || obj.url.match(/catalogLogin/))    // catalog login for holds
         ) {
           if (_originalUrl.match(/UserLogin/) || obj.url.match(/catalogLogin/)) {
-            window.location.reload();
+            var event = new CustomEvent('VuFind.lightbox.login', {
+                bubbles: true,
+                cancelable: true
+            });

```

Comment:
Should we make the other events bubble and cancelable too?

https://github.com/vufind-org/vufind/pull/620/files#diff-a7248d4d505bd3b4c0f7318f539c51d6R246